src/update_agent/actor.rs

```diff
@@ -141,8 +142,10 @@ impl UpdateAgent {
                 self.steady_interval
             }
             UpdateAgentState::UpdateStaged((_, postponements)) => {
-                // Note: no jitter if postponing reboots.
```

Comment:
I think this method outgrew its initial small logic and it's now in need of some unit tests.

I'd suggest to split out the discriminant and state check into its own function (only needing previous and current states). Then it can be tested by manually driving through states, like this other existing test: https://github.com/coreos/zincati/blob/v0.0.20/src/update_agent/mod.rs#L497

Reply:
I factored them out as independent associated functions for ease of unit testing just those two functions within `update_agent/actor.rs`.